samples/getting_started/CatStatesAndGHZ.qs

```diff
@@ -0,0 +1,63 @@
+import Std.Diagnostics.*;
+
+/// # Summary
+/// Greenberger–Horne–Zeilinger and Cat state sample
+///
+/// # Description
+/// This Q# program shows how to prepare the GHZ state in a register of 3 qubits
+/// and a generalized GHZ state (Cat state) in a register of 5 qubits.
+///
+/// # Remarks
+/// The Greenberger–Horne–Zeilinger state, or GHZ state, is a state with 3
+/// qubits defined as: |GHZ〉 = (|000〉 + |111〉) / √2.
+///
+/// The GHZ state is said to be a maximally entangled state, a multi-qubit
+/// state where the state of any one qubit is not separable from the state
+/// of any of the other qubits.
+///
+/// The generalized form of the GHZ state across any number of qubits is
+/// called a Cat state, and the GHZ state is a special case of the Cat
+/// state where the number of qubits is 3.
+///
+/// # References
+/// - [Greenberger–Horne–Zeilinger state](https://en.wikipedia.org/wiki/Greenberger%E2%80%93Horne%E2%80%93Zeilinger_state)
+operation Main() : (Result[], Result[]) {
+
+    use ghz = Qubit[3]; // Allocate 3 qubits for GHZ state.
+    PrepareGHZState(ghz); // Prepare a GHZ state in the register.
+    DumpRegister(ghz); // Show the GHZ state in the register.
+    let ghzResults = MResetEachZ(ghz); // Measure and reset qubits.
+
+    use cat5 = Qubit[5]; // Allocate 5 qubits for GHZ state.
+    PrepareCatState(cat5); // Prepare a Cat₅ state in the register.
+    DumpRegister(cat5); // Show the Cat₅ state in the register.
+    let catResults = MResetEachZ(cat5); // Measure and reset qubits.
```

Comment:
I'm not convinced there is value in doing this twice. `PrepareGHZState` should not have the assert on size 3 since GHZ states can be established on any size array, and then there is no difference between the two. I would simplify this sample to just show one of the cases, either 3 or 5, and simplify the file name and operation names here to use either GHZ or Cat. Then it can just be one line in the descriptive comment above main that clarifies that the terms are equivalent.

Reply:
Left only one sample.